part_1/src/com/chernova/libraryXML/Book.java

```diff
@@ -0,0 +1,114 @@
+package com.chernova.libraryXML;
+
+import java.util.Comparator;
+import java.util.Date;
+
+public class Book implements Comparable<Book>, Comparator<Book> {
+    private int id;
+    private String title;
+    private String text;
+    private String isbn;
+    private Genre genre;
+    private String publishDate;
+    private Date localDate;
+
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
```

Comment:
we should not have possibility to update book id. It should be generated automatically